builtin/providers/azurerm/resource_arm_route_table.go

```diff
@@ -73,7 +76,6 @@ func resourceArmRouteTable() *schema.Resource {
 
 			"subnets": {
 				Type:     schema.TypeSet,
-				Optional: true,
```

Comment:
It shouldn't be, that value can't actually be set, the code was never writing the value to the API either.